scripts/launch.sh

```
@@ -26,19 +26,26 @@ MODE="${2:-tmux}"
 CONFIG="${3:-}"
 
 # Validation
-if [ -z "$REPO" ]; then
+if [ -z "$REPO" ]; then
     echo "Usage: $0 <owner/repo> [mode] [config]"
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)